### PR TITLE
fix format change error for YUY2

### PR DESCRIPTION
**Pull request by jiteshkris**

### CODE REVIEW

**Comment by jiteshkris** [comment]
@NaveenThum is added to the review. #Closed

**Comment by jiteshkris** [comment]
@SangChoe is added to the review. #Closed

**Comment by jiteshkris** [comment]
@chrilaMSFT is added to the review. #Closed

**Comment by jiteshkris** [comment]
@kasumman is added to the review. #Closed

**Comment by jiteshkris** [comment]
@LPBourret is added to the review. #Closed

**Comment by jiteshkris** [comment]
@j-leungyy is added to the review. #Closed

**Comment by jiteshkris** [comment]
@gerritsMSFT is added to the review. #Closed

File: avstream/sampledevicemft/multipinmfthelpers.h [hunk 1]
```diff
@@ -201,10 +201,10 @@ class CPinQueue: public IUnknown{
 private:
     DWORD                m_dwInPinId;           /* This is the input pin       */
     IMFSampleList        m_sampleList;          /* List storing the samples    */
-    IMFDeviceTransform*  m_pTransform;         /* Weak reference to the the device MFT */
     GUID                 m_streamCategory;
     ULONG                m_cRef;
 protected:
+    IMFDeviceTransform* m_pTransform;         /* Weak reference to the the device MFT */
```
Comment (NaveenThum) [inline avstream/sampledevicemft/multipinmfthelpers.h:207]: >   /* Weak refe [](http://example.com/codeflow?start=43&length=15)

nit - fix alignment of /*

File: avstream/sampledevicemft/multipinmfthelpers.h [hunk 2]
```diff
@@ -473,6 +473,7 @@ class CDecoderTee : public CVideoProcTee {
     DWORD                m_lNeedInputRequest;
     GUID                 m_streamCategory; // Needed for bind flags
     BOOL                 m_bXvpAdded;
+    ComPtr<CVideoProcTee> m_spXvp;
```
Comment (NaveenThum) [inline avstream/sampledevicemft/multipinmfthelpers.h:476]: >m_spXvp [](http://example.com/codeflow?start=26&length=7)

Could this be anything but an Xvptee (e.g., decodertee)? If so, change the name of variable.

**Comment by NaveenThum** [comment]
I guess these changes are not in OS repo yet. Is the plan to mirror these changes once they go on GitHub?

File: avstream/sampledevicemft/multipinmfthelpers.h (hunk 1 above)
Comment (kasumman) [reply avstream/sampledevicemft/multipinmfthelpers.h:207]: on the member m_pTransform itself as well

File: avstream/sampledevicemft/multipinmftutils.cpp [hunk 3]
```diff
@@ -1292,7 +1292,10 @@ UpdateAllocatorAttributes(
         level = spDevice->GetFeatureLevel();
         dwBindFlags |= ((level >= D3D_FEATURE_LEVEL_10_0) ? D3D11_BIND_SHADER_RESOURCE : 0);
     }
-
+    else
```
Comment (kasumman) [inline avstream/sampledevicemft/multipinmftutils.cpp:1295]: >else [](http://example.com/codeflow?start=4&length=4)

Doesn't this make the line 1269 meaningless?
Or is D3D11_BIND_RENDER_TARGET considered as fallback value?

**Comment by jiteshkris** [comment]
> I guess these changes are not in OS repo yet. Is the plan to mirror these changes once they go on GitHub?

I will make these changes after I check this in

File: avstream/sampledevicemft/multipinmftutils.cpp (hunk 3 above)
Comment (jiteshkris) [reply avstream/sampledevicemft/multipinmftutils.cpp:1295]: It is a fallback value

File: avstream/sampledevicemft/multipinmfthelpers.h (hunk 2 above)
Comment (jiteshkris) [reply avstream/sampledevicemft/multipinmfthelpers.h:476]: I don't want to change too much.

**Pull request merged**